api/v1beta1/verticarestorepointsquery_types.go

```diff
@@ -106,6 +106,13 @@ func (vrpq *VerticaRestorePointsQuery) ExtractNamespacedName() types.NamespacedN
 	}
 }
 
+func (vrpq *VerticaRestorePointsQuery) ExtractVDBNamespacedName() types.NamespacedName {
```

Comment:
Can we use the one that was added to v1/helpers.go?

Reply:
Are you sure? It looks pretty identical:
```
func (vscr *VerticaScrutinize) ExtractVDBNamespacedName() types.NamespacedName {
	return types.NamespacedName{
		Name:      vscr.Spec.VerticaDBName,
		Namespace: vscr.ObjectMeta.Namespace,
	}
}
```

Reply:
Oh you mean the one in  v1beta1/helpers.go. I was looking into v1/helpers.go and did not find it so I was confused. Yes we an reuse that one.

Reply:
I can create a function like `genNamespacedName(name, namespace)` and have vscr and vrpq call it.

Reply:
I still don't get what's the issue with `ExtractVDBNamespacedName()`

Reply:
To be honest I got confused with the different versions of these functions. So, maybe some of the comments could have been explained away had I paid closer attention the function signature. It did feel like we were duplicating logic. I liked the generic function you ended up adding to the `names` package. We could use that for more use cases in the future.